pkgs/development/libraries/qt-6/qtModule.nix

```diff
@@ -32,9 +38,48 @@ stdenv.mkDerivation (args // {
         moveToOutput "$dir" "$dev"
       done
     fi
+    fixQtBuiltinPaths $out/lib "*.pr?"
     ${args.postInstall or ""}
   '';
 
+  preConfigure = ''
+    ${args.preConfigure or ""}
+
```

Comment:
```suggestion
  preConfigure = args.preConfigure or "" + ''
```

Reply:
This will slightly change the meaning and not guarantee a newline where it ought to be at the end

I've seen this style in nixpkgs fairly often, why change it?

Reply:
preConfigure should always be a multiline string so this should be safe.

> I've seen this style in nixpkgs fairly often, why change it?

newer things are written like this more often.

Reply:
Done!